### ISSUE TITLE
Add GET document functionality to controllers

### ISSUE THREAD

**Issue by neilshevlin** [hollymcevoy/Sweng-2022#6]
Currently there are no documents in the knowledge base. Add documents to the knowledge base and retrieve some information about the documents available. 

The client should be able to query the API about available documents in the knowledge base. A get request to GET /documents should return a list of the documents available to the user. The returned body should include the id of the documents so that the user can get the document back from the API OR make some question which refers to a specific doc.

**Comment by hollymcevoy** [comment]
Would this be able to be like a drop-down menu type of situation where they would receive a drop-down list of the available documentation?

**Comment by neilshevlin** [comment]
Yeah sure it could be something like that. You can return a singular document or multiple documents in array.

**Comment by neilshevlin** [comment]
Added in commit 78d667b

Merged to Dev
